app/api/data.py

```
@@ -0,0 +1,1157 @@
+    status: Optional[str] = Query(
+        None, description="Player status: active, injured, suspended"
+    ),
+    fantasy_relevant: Optional[bool] = Query(
+        None, description="Filter by fantasy relevance"
+    ),
+    include_stats: bool = Query(False, description="Include current season stats"),
+    include_sentiment: bool = Query(
+        False, description="Include current sentiment data"
+    ),
+    limit: int = Query(50, ge=1, le=200, description="Number of players to return"),
+    db=Depends(get_database),
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)